hs/src/Reach/Verify/SMT.hs

```diff
@@ -1344,6 +1349,20 @@ smt_n = \case
         smt_invblock (B_Assume True) inv
         smt_invblock (B_Assume False) cond
         smt_n k
+
+      insertConstrainedMaps :: S.Set DLMVar -> S.Set DLMVar
+      insertConstrainedMaps = insertConstrainedMaps' invTail
+      DLBlock _ _ invTail _ = inv
+
+      insertConstrainedMaps' :: DLTail -> S.Set DLMVar -> S.Set DLMVar
+      insertConstrainedMaps' dltail maps = case dltail of
+        DT_Return _ -> maps
+        DT_Com stmt dltail' ->
+          let tailMaps = insertConstrainedMaps' dltail' maps in
```

Comment:
We don't use `let ... in`

Reply:
I did remember that rule, but I wonder why we don't use it? Eg here there seems to be no better way to avoid code repetition.

Reply:
It makes the indentation weird, so we use `do` or `where` instead